client/my-sites/pages/page/index.js

```diff
@@ -200,26 +199,26 @@ class Page extends Component {
 		}
 
 		if ( this.props.page.status !== 'trash' ) {
-			return createFragment( {
-				separator: <MenuSeparator />,
-				item: (
+			return (
+				<div>
+					<MenuSeparator />
 					<PopoverMenuItem className="page__trash-item" onClick={ this.updateStatusTrash }>
 						<Gridicon icon="trash" size={ 18 } />
 						{ this.props.translate( 'Trash' ) }
 					</PopoverMenuItem>
-				),
-			} );
+				</div>
+			);
 		}
 
-		return createFragment( {
-			separator: <MenuSeparator />,
-			item: (
+		return (
+			<div>
```

Comment:
Is there any _advantage_ to adding a wrapping `div` over returning an array and specifying keys for each element? Seems like it could be nice to avoid an unnecessary DOM element.

Reply:
Yeah, fair enough. I was trying to avoid arrays altogether, but I guess there's no harm in keeping them.

Reply:
f015806